spyder/preferences/languageserver.py

```diff
@@ -120,33 +120,33 @@ def __init__(self, parent, language=None, cmd='', host='127.0.0.1',
         super(LSPServerEditor, self).__init__(parent)
 
         description = _(
-            'To create a new server configuration, you need to select a '
-            'programming language and the server hostname/IP and port to '
-            'use. If starting a local server, also provide the command to '
-            'execute it and any arguments that should be passed on startup.'
-            '<br><br>'
-            'You can use the placeholders <tt>{host}</tt> and '
-            '<tt>{port}</tt> in the server arguments field to '
-            'automatically fill in the respective values.<br>'
+            "To create a new server configuration, you need to select a "
+            "programming language, set the command to start its associated "
+            "server and any arguments that should be passed to it on "
+            "startup. Additionally, you can set the server's hostname and "
+            "port, if using a tcp connection to it or you want to "
```

Comment:
Correct caps and a little clearer.

```suggestion
            "port, if not connecting to a local server using pipes instead of TCP."
```

Reply:
I don't like this suggestion. I think it's better to be explicit here: you need to pass a host and port if you want to connect to an external server *or* use tcp instead of stdio.

Reply:
Okay, then how about

```suggestion
            "port if connecting to an external server, "
            "or to a local one using TCP instead of stdio pipes."
```

Reply:
Much better, thanks!